production/kubernetes/install-bare.sh

```diff
@@ -25,7 +25,7 @@ check_installed() {
 check_installed curl
 check_installed envsubst
 
-MANIFEST_BRANCH=v0.26.1
+MANIFEST_BRANCH=v0.27.0
```

Comment:
I wonder if this should actually be release-0.27 given our new branch convention?

Reply:
I interpreted it that way. It looks like it will pull from the release branch, not the rc tag.

Reply:
This will resolve the URL below to https://raw.githubusercontent.com/grafana/agent/v0.27.0/production/kubernetes/agent-bare.yaml 

I think `MANIFEST_BRANCH` should be changed to either be the literal branch name `release-v0.27` or the tag `v0.27.0-rc.0`